src/index.html

```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
-                            <li>Complete a <a href="files/EVA_Submission_template.METADATA_TEMPLATE_VERSION.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. Please provide as much metadata as possible since this information is extremely useful for downstream analysis and is directly related to the frequency at which datasets archived at EVA are reused. For reference, here is an <a href="files/EVA_Submission_template.METADATA_TEMPLATE_VERSION_mockup.xlsx" target="_blank"><b>example</b></a> of a completed metadata template.</li>
-                        </ul>
-                        <p>Please also note that the template requires the submitter to fill in some personal data, which will be used as described in our <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/eva-submission-service" target="_blank">privacy notice</a>.</p>
-                    </div>
-
-                    <h4 id="submit-contact"><span class="icon icon-generic" data-icon="C" aria-hidden="true"></span> Contact</h4>
-                    <div>
-                        <p>Contact  <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> to request a submission. You will receive a custom private FTP account to deposit your data.</p>
-                    </div>
-
-                    <h4 id="submit-submit"><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
-                    <div>
-                        <p>Upload your VCF file(s), metadata template and any associated data file(s) to your private FTP location.</p>
+                    <div class="submit-data-diagram">
+                        <div class="box green">
+                            <h3>📄 Prepare your data and metadata</h3>
+                            <ul>
+                                <li>Prepare valid VCF file(s), which can be validated prior to submission using the <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>.</li>
+                                <li>Complete a <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_template.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Please provide as much metadata as possible since this information is useful for downstream analysis and leads to increased dataset reuse. For reference, here is an <a href="https://www.ebi.ac.uk/eva/files/EVA_Submission_template.V1.1.5_mockup.xlsx" target="_blank"><b>example</b></a> of a completed metadata template.</li>

```

Comment:
```suggestion
                                <li>Complete a <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_template.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Please provide as much metadata as possible since this information is useful for downstream analysis and leads to increased dataset reuse. For reference, here is an <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_Example.xlsx" target="_blank">example</a> of a completed metadata template.</li>

```
This is the updated example. Also not sure it should be bold since nothing else in the box is...

Reply:
updated.